base/system/format/lang/ja-JP.rc

```diff
@@ -31,7 +31,7 @@ BEGIN
     STRING_FMIFS_FAIL "FMIFS エントリ ポイント を設定できませんでした。\n\n"
     STRING_UNKNOW_ARG "不明な引数: %s\n"
     STRING_DRIVE_PARM "必要なドライブを指定するパラメータが不足しています。\n\n"
-    STRING_ERROR_DRIVE_TYPE "ドライブの種類がわかりませんでした"
+    STRING_ERROR_DRIVE_TYPE "ドライブの種類がわかりませんでした.\n"
```

Comment:
Needs also the round circle full stop character (see the line above)

Reply:
Done.